docs/documentation/maya/muscle.md

```diff
@@ -111,108 +70,117 @@ Following, we present the key aspects to keep in mind while painting each attrib
+ - **Gravity Direction** (Float3, {0.0. -1.0, 0.0}): Sets the direction of the gravity acceleration.
+    - Vectors introduced do not need to be normalized, but they will get normalized internally.
 
 ### Advanced Settings
 
 #### Stiffness Settings
  - **Use Custom Stiffness** (Boolean, False): Toggles the use of a custom stiffness value. If enabled, the Material is ignored and the Stiffness parameter is used instead.
-    - If we use a custom stiffness, **Material** and **Stiffness Multiplier** will be disabled and **Stiffness** will be used instead.
+    - If we use a custom stiffness, *Material* and *Stiffness Multiplier* will be disabled and *Stiffness* will be used instead.
 
 #### Dynamic Properties
  - **Global Damping** (Float, 0.75): Sets the scaling factor applied to the global damping of every point.
-    - Has a range of \[0.0, 1.0\] [^1]
+    - Has a range of \[0.0, 1.0\]. Upper limit is soft, higher values can be used.
  - **Inertia Damper** (Float, 0.0): Sets the linear damping applied to the dynamics of every point.
-    - Has a range of \[0.0, 1.0\] [^1]
+    - Has a range of \[0.0, 1.0\]. Upper limit is soft, higher values can be used.
  - **Rest Length Multiplier** (Float, 1.0): Sets the scaling factor applied to the edge lengths at rest.
-    - Has a range of \[0.0, 2.0\] [^1]
+    - Has a range of \[0.0, 2.0\]. Upper limit is soft, higher values can be used.
  - **Compression Multiplier** (Float, 1.0): Sets the scaling factor applied to the compression resistance of every point.
-    - Has a range of \[0.0, 2.0\] [^1]
+    - Has a range of \[0.0, 2.0\]. Upper limit is soft, higher values can be used.
  - **Stretching Multiplier** (Float, 1.0): Sets the scaling factor applied to the stretching resistance of every point.
-    - Has a range of \[0.0, 2.0\] [^1]
+    - Has a range of \[0.0, 2.0\]. Upper limit is soft, higher values can be used.
  - **Attenuation Velocity factor** (Float, 1.0): Sets the weight of the attenuation applied to the whole simulation driven by the Attenuation Matrix.
-    - Has a range of \[0.0, 10.0\] [^1]
+    - Has a range of \[0.0, 10.0\]. Upper limit is soft, higher values can be used.
  - **Hard Attachments** (Boolean, True): If enabled, attachment constraints will force the vertices to stick to target transformation completely.
 
+### Debug attributes
+ - **Debug** (Boolean, False): Enable or Disable the debug functionalities in the viewport for the AdnRibbonMuscle deformer.
+ - **Feature** (Enumerator, "Muscle Fibers"): A list of debuggable features for this deformer.
+     - Muscle Fibers: Draw *Muscle Fibers* fiber directions on the simulated mesh's surface.
+     - Attachment Constraints: Draw *Attachment Constraints* connections from the simulated mesh to the attachments.
+     - Sliding On Segment: Draw *Sliding On Segment* connections from the simulated mesh to the segment the simulated mesh is sliding on.
```

Comment:
Open dicussion. We expose sliding on segment here but we always refere to slide on segment. We might want to change this @carlosmonteagudoinbibo

Reply:
@josevalverdeinbibo @enriquemeseguerinbibo @luisalbaceteinbibo @carlosmonteagudoinbibo 

I'm adding pending TODO's in this PR as I dom't have the resources at the moment to update screenshots
- Update Paint Tool UI Screenshots including Global Damping
- Update the Skin Deformer image with the image provide by Enrique
- We should maybe add screenshots for SOS constraints?
- Maybe add an example for the edge evaluator?

Reply:
We agreed to rename it to "Slide on Segment"